provider/source/src/properties/uprops_serde.rs

```diff
@@ -64,34 +64,17 @@ pub(crate) mod enumerated {
     }
 }
 
+#[cfg(any(feature = "use_wasm", feature = "use_icu4c"))]
 pub(crate) mod code_point_prop {
-    #[derive(serde::Deserialize)]
```

Comment:
Issue: again, cldr_serde is the one place where I think we should keep the whole struct. Also, it's not completely dead code: it makes the Deserialize impls check that those fields are well formed.

Reply:
eh, I don't think we generally assert fields that we don't use, I don't think it's that valuable